doc/media/openage/modpacks.md

```diff
@@ -0,0 +1,176 @@
+* Descriptions and author information
+
+Furthermore, the file can specify the interaction with other modpacks:
+
+* Dependencies on other modpacks
+* Conflicts with other modpacks
+* Replacment of other modpacks
+* Specification of a preferred load order (in relation to other mods)
+
+A full list of configuration parameters can be found [here](modpack_definition_file.md).
+
+
+## Distribution
+
+Modpacks can be distributed as standalone packages or via a package
+repository. Each of these methods has their own advantages.
+
+
+### Standalone Distribution
+
+Any modpack can be distributed as a standalone package by publishing
+a ZIP or tar.gz archive of the modpack on a hosting platform or a
+website. The archive can be imported by the openage launcher.
+
+It is the easiest way to publish a modpack for openage. However, updates
+and dependencies have to be done manually. Therefore, it is recommended
+to use a package repository to offer better convenience features for users.
+
+A note of advice: Allow everyone to download older versions of a modpack.
+Other modpacks might depend on a specific release version of your content.
+
+
+### Package Repositories
+
+A package repository (repo for short) is a dedicated hosting platform for
+modpacks. They are inspired by the package management systems of Linux
+distributions, Python, Rust and others. Package repositories have the following
+advantages:
+
+* Can be checked regularly for updates
+* Automatically download and install dependencies if they are in the same repo
+* Can store the history of modpacks (previous versions)
+* They can be searched by using keywords or content filters
+* Can sign modpacks to indicate trust (i.e. check for exploits or malware)
+
+Everyone can set up a package repository. Users can add an unlimted number of
+repositories in the openage launcher. Some repositories are added by default
+when openage is installed.
+
+Repositories are the preferred method of distributing mods.
+
+
+### Licensing
+
+Once a mod is out in the open, it's good to clarify how and what other creators are
+allowed to do with it. This can be done by adding a **license** to the
+modpack. Using a license is completely optional, but it makes sense to
+add one in some cases. For example, a license can specify how to give credit
+or whether files from a modpack are allowed to be integrated into other modpacks.
+
+The license can be chosen for every modpack individually by the creator.
+While the engine is licensed as GPLv3+, modpack licenses do not need to
+be compatible with it. Repos may decide which licenses they allow and
+reject. Additionally, they may offer a default license.
+
+When you include (not reference) assets from modpacks of other creators,
+always check their license first. Copying files may be subject to terms set
+in the license. It is usually safer (and less frowned upon) to have the modpack
+referenced as a dependency than to copy files over.
```

Comment:
the license section should be about modpacks in general and not in the repos section.

Reply:
Split off into own section.